.circleci/config.yml

```diff
@@ -127,8 +141,101 @@ jobs:
             git checkout master
             export GITHUB_TOKEN=${CLOUD_MTA_BOT_GITHUB_TOKEN}
             curl -sL https://git.io/goreleaser | bash
-            
-  publish:
+
+  publish-to-dockerhub:
+    docker:
+      - image: cimg/go:1.17
+    working_directory: ~/go/src/github.com/SAP/cloud-mta-build-tool
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 20.10.6
+      - run:
+          name: build image pre-setup
+          command: |
+            #Make sure HEAD points to master
+            git checkout master
+            git fetch
+            git rebase
+      # - run:
```

Comment:
don't leave commented out code as part of the source code

Reply:
Done;

Delete java 8, 11 and node 14, 18 combination docker image, github container, npm publish code, and the redundant workflow